.github/release.yml

```diff
@@ -0,0 +1,18 @@
+changelog:
```

Comment:
Super simple, just include this template in the .github folder, and it will pick it up and use it.  It just captures PRs merged since the last release like I do now, and categorizes them by label, like I do now.  It also adds some extra stuff like "first time contributors" and a link to the full changelog.